src/sage/schemes/plane_conics/con_field.py

```diff
@@ -1063,79 +1067,82 @@ def rational_point(self, algorithm='default', read_cache=True):
+            sage: C.defining_polynomial()(list(p))
             0
 
-            sage: K.<i> = QuadraticField(-1)                                            # optional - sage.rings.number_field
-            sage: D = Conic(K, [3, 2, 5])                                               # optional - sage.rings.number_field
-            sage: D.rational_point(algorithm = 'rnfisnorm') # output is random          # optional - sage.rings.number_field
+            sage: K.<i> = QuadraticField(-1)                                            # needs sage.rings.number_field
+            sage: D = Conic(K, [3, 2, 5])                                               # needs sage.rings.number_field
+            sage: D.rational_point(algorithm='rnfisnorm')  # output is random           # needs sage.rings.number_field
             (-3 : 4*i : 1)
 
-            sage: L.<s> = QuadraticField(2)                                             # optional - sage.rings.number_field
-            sage: Conic(QQ, [1, 1, -3]).has_rational_point()                            # optional - sage.rings.number_field
+            sage: # needs sage.rings.number_field
+            sage: L.<s> = QuadraticField(2)
+            sage: Conic(QQ, [1, 1, -3]).has_rational_point()
             False
-            sage: E = Conic(L, [1, 1, -3])                                              # optional - sage.rings.number_field
-            sage: E.rational_point()                        # output is random          # optional - sage.rings.number_field
+            sage: E = Conic(L, [1, 1, -3])
+            sage: E.rational_point()                        # output is random
             (-1 : -s : 1)
 
         Currently Magma is better at solving conics over number fields than
         Sage, so it helps to use the algorithm 'magma' if Magma is installed::
 
-            sage: q = C.rational_point(algorithm='magma',           # optional - magma  # optional - sage.rings.number_field
+            sage: # optional - magma, needs sage.rings.number_field
+            sage: q = C.rational_point(algorithm='magma',
             ....:                      read_cache=False)
-            sage: q                       # output is random,       # optional - magma  # optional - sage.rings.number_field
+            sage: q                       # output is random
             (1/5*b^2 : 1/5*b^2 : 1)
-            sage: C.defining_polynomial()(list(q))                  # optional - magma  # optional - sage.rings.number_field
+            sage: C.defining_polynomial()(list(q))
             0
-            sage: len(str(p)) > 1.5*len(str(q))                     # optional - magma  # optional - sage.rings.number_field
+            sage: len(str(p)) > 1.5*len(str(q))
             True
-
-            sage: D.rational_point(algorithm='magma',       # random, optional - magma  # optional - sage.rings.number_field
+            sage: D.rational_point(algorithm='magma',
             (1 : 2*i : 1)
```

Comment:
Perhaps we should keep `random`. Here and below.

Reply:
done in 278e3bea772